dev/com.ibm.ws.session/src/com/ibm/ws/session/HttpSessionFacade.java

```diff
@@ -10,105 +10,22 @@
  *******************************************************************************/
 package com.ibm.ws.session;
 
-import java.io.IOException;
-import java.io.ObjectInput;
-import java.io.ObjectOutput;
-import java.util.Enumeration;
-
-import javax.servlet.ServletContext;
 import javax.servlet.http.HttpSession;
 import javax.servlet.http.HttpSessionContext;
 
-import com.ibm.websphere.servlet.session.IBMApplicationSession;
-import com.ibm.websphere.servlet.session.IBMSession;
-import com.ibm.wsspi.servlet.session.IBMSessionExt;
+/*
+ * Servlet 6.0 update: Specific API for session up to servlet 5.0
+ * Common methods are moved to AbstractHttpSessionFacade
+ */
 
-public class HttpSessionFacade implements IBMSession, IBMSessionExt { // cmd 196151
+public class HttpSessionFacade extends AbstractHttpSessionFacade { // cmd 196151
 
-    protected transient SessionData _session = null;
     private static final long serialVersionUID = 3108339284895967670L;
 
-    public HttpSessionFacade(SessionData data) {
+    public HttpSessionFacade(AbstractSessionData data) {
```

Comment:
Could this stay SessionData?  That is the AbstractSessionData that goes with this class right?